frontend-react/src/components/Content/PageGenerationTools.tsx

```diff
@@ -2,13 +2,14 @@
 
 import { CrumbConfig, WithCrumbs } from "../Crumbs";
 
+type ElementFunction = () => JSX.Element;
 export const contentContainer = (
-    content: JSX.Element,
+    content: JSX.Element | ElementFunction,
     crumbs: CrumbConfig[]
 ) => {
     const wrappedElement = (
         <div className="grid-container rs-documentation usa-prose">
-            {content}
+            {content instanceof Function ? content() : content}
```

Comment:
is the hope that once we've moved off of the legacy system that we can remove this complexity? if so, can we leave a comment to that effect?

Reply:
Hmm... 🤔 I actually may have set us up to _only_ need the `content()` bit anyway. Fancy that!